patches/chromium/fix_allow_guest_webcontents_to_enter_fullscreen.patch

```diff
@@ -0,0 +1,20 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Samuel Attard <sattard@salesforce.com>
+Date: Mon, 6 Jun 2022 14:25:15 -0700
+Subject: fix: allow guest webcontents to enter fullscreen
+
+This can be upstreamed, a guest webcontents can't technically become the focused webContents. This DCHECK should allow all guest webContents to request fullscreen entrance.
```

Comment:
Have you already upstreamed this? Would love to have the codereview link!

Reply:
Haven't had a chance yet